user.py

```diff
@@ -0,0 +1,20 @@
+class User:
+    email = 'no@no.com'
+    def __init__(name, age):
+        self.name = name
+        self.age = age
+
+    def display_from_file(filename):
+        f = open(filename, 'r')
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *f*. The resource is allocated by call *builtins.open*. Execution paths that do not contain closure statements were detected. To prevent this resource leak, close *f* in a try-finally block or declare it using a `with` statement.

**More info**
[View details about the `with` statement in the Python developer&#39;s guide](https://www.python.org/dev/peps/pep-0343/) (external link).

Reply:
Ok, I agree, my python sucks 👍